src/pages/index.js

```diff
@@ -97,9 +228,13 @@ addButton.addEventListener("click", () => {
   addCardModal.open();
 });
 
-profileEditButton.addEventListener("click", () => {
-  const userData = profileInfo.getUserInfo();
-  profileTitleInput.value = userData.title;
-  profileDescriptionInput.value = userData.description;
+profileEditButton.addEventListener("click", async () => {
+  const userData = await api.loadUserInfo();
```

Comment:
`  const userData = await api.loadUserInfo();
`

Please, delete it


You have `userInfo.getUserInfo();` for getting the profile data not to waste resources on fetching it from the server


```js
  const userData = userInfo.getUserInfo();
  profileTitleInput.value = userData.title;
  profileDescriptionInput.value = userData.description;

```